tests/functional/ContentRepositoryTest.php

```diff
@@ -969,6 +1032,12 @@ public function it_checks_existence_of_lang_code_on_translations_join()
             [['translations.title', 'DESC']],
             null
         );
+
+        $this->repository->getContentsByLevel(
```

Comment:
It should be moved to separate test because @expectedException annotation will catch only one error